.github/workflows/widget_catalog_app_distributtion.yml

```diff
@@ -0,0 +1,54 @@
+name: Interactive Diary Design System CD
+
+on:
+  push:
+    branches:
+      - feat-CD-widget-catalog
+#    paths:
+#      - interactive_diary/packages/nartus_ui_package/**
+
+jobs:
+  distribute:
+    name: "Widget Catalog app Firebase Distribution"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/nartus_ui_package/example
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "11"
+      - name: Install Flutter
+        uses: subosito/flutter-action@v2.6.1
+        with:
+          flutter-version: "3.3.2"
+          channel: "stable"
+      - name: Flutter install dependencies
+        shell: bash
+        run: |
+          flutter clean
+          flutter pub get
+      - name: Build Android apk
+        run: flutter build apk --release
+      # - name: Build IOS bundle
+      #   run: flutter build ipa --debug --no-codesign
+      # - name: Upload iOS artifact
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     name: interactive_diary.app
+      #     path: /Users/runner/work/interactive_diary/interactive_diary/packages/nartus_ui_package/example/build/ios/iphoneos/Runner.app
+      - name: Upload Android artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: app-debug.apk
+          path: /Users/runner/work/interactive_diary/interactive_diary/packages/nartus_ui_package/example/build/app/outputs/flutter-apk/app-release.apk
```

Comment:
try to use relative path, like 'packages/nartus_ui_package/example/build/app/outputs/flutter-apk/app-release.apk'?